examples/hybrid/sphere/hadley_circulation.jl

```diff
@@ -0,0 +1,332 @@
+using Test
+using LinearAlgebra
+
+import ClimaCore:
+    ClimaCore,
+    slab,
+    Spaces,
+    Domains,
+    Meshes,
+    Geometry,
+    Topologies,
+    Spaces,
+    Fields,
+    Operators
+import ClimaCore.Utilities: half
+
+using OrdinaryDiffEq: ODEProblem, solve
+using DiffEqBase
+using ClimaTimeSteppers
+
+using NCDatasets, ClimaCoreTempestRemap
+using Statistics: mean
+
+import Logging
+import TerminalLoggers
+Logging.global_logger(TerminalLoggers.TerminalLogger())
+
+# 3D Hadley-like Meridional Circulation (DCMIP 2012 Test 1-2)
+# Reference: http://www-personal.umich.edu/~cjablono/DCMIP-2012_TestCaseDocument_v1.7.pdf, Section 1.2
+
+
+# visualization artifacts
+ENV["GKSwstype"] = "nul"
+using ClimaCorePlots, Plots
+Plots.GRBackend()
+dir = "hadley_circulation"
+path = joinpath(@__DIR__, "output", dir)
+mkpath(path)
+
+function linkfig(figpath, alt = "")
+    # buildkite-agent upload figpath
+    # link figure in logs if we are running on CI
+    if get(ENV, "BUILDKITE", "") == "true"
+        artifact_url = "artifact://$figpath"
+        print("\033]1338;url='$(artifact_url)';alt='$(alt)'\a\n")
+    end
+end
+
+
+# set up function space
+function sphere_3D(
+    R = 6.37122e6,
+    zlim = (0, 12.0e3),
+    helem = 6,
+    zelem = 30,
+    npoly = 4,
+)
+    FT = Float64
+    vertdomain = Domains.IntervalDomain(
+        Geometry.ZPoint{FT}(zlim[1]),
+        Geometry.ZPoint{FT}(zlim[2]);
+        boundary_names = (:bottom, :top),
+    )
+    vertmesh = Meshes.IntervalMesh(vertdomain, nelems = zelem)
+    vert_center_space = Spaces.CenterFiniteDifferenceSpace(vertmesh)
+
+    horzdomain = Domains.SphereDomain(R)
+    horzmesh = Meshes.EquiangularCubedSphere(horzdomain, helem)
+    horztopology = Topologies.Topology2D(horzmesh)
+    quad = Spaces.Quadratures.GLL{npoly + 1}()
+    horzspace = Spaces.SpectralElementSpace2D(horztopology, quad)
+
+    hv_center_space =
+        Spaces.ExtrudedFiniteDifferenceSpace(horzspace, vert_center_space)
+    hv_face_space = Spaces.FaceExtrudedFiniteDifferenceSpace(hv_center_space)
+    return (hv_center_space, hv_face_space)
+end
+
+# RHS tendency specification
+function rhs!(dydt, y, parameters, t, alpha, beta)
+
+    (; coords, face_coords, τ, u₀, ω₀, K, ρ₀, ystar) = parameters
+
+    ϕ = coords.lat
+    zc = coords.z
+    zf = face_coords.z
+    ϕf = face_coords.lat
+
+    uu = @. u₀ * cosd(ϕ)
+    uv = @. -(R * ω₀ * pi * ρ₀ / K / z_top / ρ_ref(zc)) *
+       sind(K * ϕ) *
+       cosd(ϕ) *
+       cosd(pi * zc / z_top) *
+       cos(pi * t / τ)
+    ω = @. (ω₀ * ρ₀ / K / ρ_ref(zf)) *
```

Comment:
```suggestion
    uw = @. (ω₀ * ρ₀ / K / ρ_ref(zf)) *
```

Reply:
@szy21 I removed the `uw = @. -ω / ρ_ref(zf) / grav` as suggested, but I wonder if we should have kept the minus sign here though?

Reply:
Oh, I see, we're not using (38) at all